src/server/scripts/Outland/BlackTemple/boss_warlord_najentus.cpp

```diff
@@ -64,6 +70,10 @@ struct boss_najentus : public BossAI
 
         BossAI::JustEngagedWith(who);
         Talk(SAY_AGGRO);
+        me->GetMap()->DoForAllPlayers([&](Player* player)
+        {
+            player->DestroyItemCount(ITEM_NAJENTUS_SPINE, 5, true);
```

Comment:
Can you get more than 5?

Reply:
Not without mangling the item template, no.